test/SourceKit/Indexing/index_effective_access_level.swift

```diff
@@ -0,0 +1,37 @@
+// RUN: %sourcekitd-test -req=index %s -- -Xfrontend -serialize-diagnostics-path -Xfrontend %t.dia %s | %sed_clean > %t.response
+// RUN: %diff -u %s.response %t.response
+
+public enum PublicEn {
+    case a
+    case b
+}
+
+enum InternalEn {
+    case a
+    case b
+}
+
+private enum PrivateEn {
+    case a
+    case b
+}
+
+public extension PublicEn {
+    func puFoo() {}
+}
+
+extension PublicEn {
+    public func puFoo() {}
```

Comment:
Did you intend to have two conflicting declarations?

Reply:
They were supposed to have different names -- fixed it. But making two extensions of the same level was intended indeed.